mods/dye/init.lua

```diff
@@ -73,6 +73,14 @@ minetest.register_craft({
 	recipe = {"group:coal"},
 })
 
+-- Manually add huckleberries->violet dye
+
+minetest.register_craft({
+	type = "shapeless",
+	output = "dye:violet 2",
```

Comment:
If a flower gives 4 dye i think a bunch of berries certainly can give 4.

Reply:
This will actually mean it's 4 * number of leaves on the bush, and they regrow much easier than flowers. I think 2 is better.

Reply:
I'm ok with either.